### PR TITLE
[Test only] include trt in cl docker

### PR DESCRIPTION
**Pull request by wushirong**
Summary:
[TEST ONLY]
This diff is testing whether install trt can help resolve header not found issue on another diff.

Test Plan: cl

Differential Revision: D32802182

### CODE REVIEW

File: .circleci/docker/common/install_tensorrt.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ -n "$TENSORRT" ]; then
```
Comment (malfet) [inline .circleci/docker/common/install_tensorrt.sh:3]:
```suggestion
if [ -n "$TENSORRT_VERSION" ]; then
```

File: .circleci/docker/common/install_tensorrt.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    python3 -m pip install --upgrade setuptools pip
+    python3 -m pip install nvidia-pyindex
+    python3 -m pip install nvidia-tensorrt==8.0.1.6 --extra-index-url https://pypi.ngc.nvidia.com
```
Comment (malfet) [inline .circleci/docker/common/install_tensorrt.sh:6]:
```suggestion
    python3 -m pip install nvidia-tensorrt==${TENSORRT_VERSION} --extra-index-url https://pypi.ngc.nvidia.com
```

File: .circleci/docker/ubuntu-cuda/Dockerfile [hunk 3]
```diff
@@ -67,6 +67,11 @@ ADD ./common/install_openssl.sh install_openssl.sh
 ENV OPENSSL_ROOT_DIR /opt/openssl
 RUN bash ./install_openssl.sh
 
+# (optional) Install TensorRT
+ARG TENSORRT
```
Comment (malfet) [inline .circleci/docker/ubuntu-cuda/Dockerfile:71]:
```suggestion
ARG TENSORRT_VERSION
```

File: test/run_test.py [hunk 4]
```diff
@@ -85,7 +85,7 @@ def skip_test_p(name: str) -> bool:
         'bottleneck_test',
         'custom_backend',
         'custom_operator',
-        'fx',        # executed by test_fx.py
+        'fx/',        # executed by test_fx.py
```
Comment (malfet) [inline test/run_test.py:88]: Can you please explain why those changes are needed?

**Review by malfet — CHANGES_REQUESTED** [review]

File: test/run_test.py (hunk 4 above)
Comment (wushirong) [reply test/run_test.py:88]: > 

in dir caffe2/test, there are dir with name "fx", "fx_acc", "fx2trt", this change is to only block test under dir caffe2/test/fx/...

**Pull request closed without merging**